assets/js/ledyer-checkout-for-woocommerce.js

```diff
@@ -392,6 +390,7 @@ jQuery(function ($) {
                                         // Ledyer will respond with a new event when order is complete
                                         // So don't redirect just yet
                                         lco_wc.isValidating = false;
+                                        lco_wc.redirectUrl = url.toString();
```

Comment:
If client validation was triggered, we run this function, `placeLedyerOrder`, which creates and validates the woo order.

Later on, after validation has been sent to Ledyer, Ledyer Checkout will trigger the event `ledyerCheckoutOrderComplete` to give the final OK. That function previously would always trigger `placeLedyerOrder`, since we initially didn't have the validation step and that was the only place to call `placeLedyerOrder`.

If validation was triggered however, we don't wanna run this function twice (as it will create a duplicate), so let's save the redirect url and when the order complete event is received, redirect immediately.